packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -23,6 +22,7 @@ export const {
 } = require('ReactFeatureFlags');
```

Comment:
The rest of these, are they still relevant to be dynamic or can we just turn them on or off?

Reply:
Only `debugRenderPhaseSideEffects` is currently dynamically configured internally.

We always set `debugRenderPhaseSideEffectsForStrictMode` to true.

Reply:
Looks like `debugRenderPhaseSideEffects ` is a dynamic value, `debugRenderPhaseSideEffectsForStrictMode` is being set to true in `InitializeReactFeatureFlags`, and nothing is changing the default value of `warnAboutDeprecatedLifecycles`.